python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1135,6 +1135,91 @@ def plot_gnss_cn0(self):
 
         self._add_figure(name='gnss_cn0', figure=figure, title='GNSS C/N0 vs. Time')
 
+    def plot_gnss_azimuth_elevation(self):
+        """!
+        @brief Plot GNSS azimuth/elevation angles.
+        """
+        result = self.reader.read(message_types=GNSSSatelliteMessage, **self.params)
+        data = result[GNSSSatelliteMessage.MESSAGE_TYPE]
+
+        if len(data.p1_time) == 0:
+            self.logger.info('No satellite data available. Skipping azimuth/elevation plot.')
+            return
+
+        # Set up the figure.
+        figure = make_subplots(
+            rows=4, cols=1,  print_grid=False, shared_xaxes=True,
```

Comment:
Any particular reason to use 4 rows and a spec below instead of just 2 rows and no `specs=` specifier? Doing it this way complicates things: you need the extra title none entries, the add trace call uses row 3 instead of 2, etc.